### PR TITLE
Removed unused variables

### PR DESCRIPTION
**Pull request by dwnoble**

### CODE REVIEW

**Review by juliawu — APPROVED** [review]
Thanks for the cleanup!

File: experimental/sdg-tracker/src/state/index.ts [hunk 1]
```diff
@@ -34,7 +34,6 @@ import rootTopics from "../config/rootTopics.json";
 import sidebarConfig from "../config/sidebar.json";
 import targetText from "../config/targetText.json";
 import {
-  CONTINENTS,
```
Comment (keyurva) [inline experimental/sdg-tracker/src/state/index.ts:37]: The declaration can also be removed: https://github.com/datacommonsorg/website/blob/master/experimental/sdg-tracker/src/utils/constants.ts#L58

**Review by keyurva — APPROVED** [review]
Thanks Dan.

File: experimental/sdg-tracker/src/state/index.ts (hunk 1 above)
Comment (dwnoble) [reply experimental/sdg-tracker/src/state/index.ts:37]: done!

**Pull request merged**